collector/filesystem_linux.go

```diff
@@ -135,8 +135,8 @@ func stuckMountWatcher(mountPoint string, success chan struct{}) {
 
 func mountPointDetails() ([]filesystemLabels, error) {
 	file, err := os.Open(procFilePath("1/mounts"))
-	if os.IsNotExist(err) {
-		// Fallback to `/proc/mounts` if `/proc/1/mounts` is missing due hidepid.
+	if os.IsPermission(err) {
```

Comment:
I was actually thinking about changing this to try fallback on any error. `if err != nil {`. What do you think?

Reply:
Checking for *any* error is the best solution; good point!

Reply:
Ping, do you want to update this?